SDK/Plugin_Attenuator.cpp

```diff
@@ -0,0 +1,142 @@
+#include "UnityAudioPluginInterface.hpp"
+
+#include <cmath>
+#include <vector>
+
+namespace attenuator
+{
+
+namespace dsp
+{
+
+static float db_to_ratio(float dB)
```

Comment:
I'd like to avoid splitting the example projects into multiple files. Ideally they should be self-contained mini demos that demonstrate the SDK to users in the simplest way possible. Having each example in a single file makes it very easy for users to digest them.